.azure-pipelines/azure-pipelines-win.yml

```diff
@@ -8,14 +8,14 @@ jobs:
     vmImage: vs2017-win2016
```

Comment:
this shows that you're not re-rendering properly or something else is wrong.

This should say what you have configured in the conda-forge yaml config file.